packages/react-composites/package.json

```diff
@@ -85,6 +85,7 @@
     "@babel/cli": "~7.16.0",
     "@babel/core": "~7.16.0",
     "@babel/preset-env": "7.13.10",
+    "@internal/fake-backends": "1.3.1-beta.1",
```

Comment:
is this versioning correct? are we making it all consistent? or should this be 1.0.0?

Reply:
Yes we are making it all consistent.

Reply:
Always keep all packlets consistent, for now.